### ISSUE TITLE
hide events

### ISSUE THREAD

**Issue by dchinmay2** [diamondburned/gotktrix#27]
Would it be possible to hide room joining/leaving events? IRC rooms have a lot of these

![image](https://user-images.githubusercontent.com/36493671/156313220-c9352874-7d15-462d-86c0-494e1324fff2.png)

**Comment by diamondburned** [comment]
Yes, but it would leave most of the room empty since gotktrix doesn't attempt to fetch the backlog automatically.

The message paginator is also broken (but no tracking issues were made yet), so it's not a good idea to try to make it fetch the backlog either.
